src/smartlock.sh

```diff
@@ -51,33 +52,64 @@ while (( $# > 0 )); do
 done
 
 if [[ -z "$LOCK" ]]; then
-    if [[ -f /home/pi/smartlock.id ]]; then
-        LOCK="$(cat /home/pi/smartlock.id)"
+    if [[ -f ${BASE_PATH}/smartlock.id ]]; then
+        LOCK="$(cat ${BASE_PATH}/smartlock.id)"
     else
         echo "ERROR: must supply a valid lock identifier to --lock"
         exit
     fi
 fi
 
+ID_CA=${cert_dir}/id_ca/identity_ca.pem
+ID_CERT=${cert_dir}/${LOCK}/identity.pem
+ID_PKEY=${cert_dir}/${LOCK}/identity_key.pem
+PERM_CA=${cert_dir}/perm_ca/permissions_ca.pem
+PERM_GOV=${cert_dir}/governance.xml.p7s
+PERM_PERMS=${cert_dir}/${LOCK}/permissions.xml.p7s
+
 if (( $SECURITY )); then
     SECURITY_ARGS=" \
     -DCPSSecurityDebug bookkeeping \
     -DCPSSecurity 1 \
-	-ID_CA ${cert_dir}/id_ca/identity_ca_cert.pem \
-	-ID_CERT ${cert_dir}/${LOCK}/${LOCK}_cert.pem \
-	-ID_PKEY ${cert_dir}/${LOCK}/private_key.pem \
-	-PERM_CA ${cert_dir}/perm_ca/permissions_ca_cert.pem \
-	-PERM_GOV ${cert_dir}/gov_signed.p7s \
-	-PERM_PERMS ${cert_dir}/${LOCK}/house1_signed.p7s \
+    -ID_CA ${ID_CA} \
+    -ID_CERT ${ID_CERT} \
+    -ID_PKEY ${ID_PKEY} \
+    -PERM_CA ${PERM_CA} \
+    -PERM_GOV ${PERM_GOV} \
+    -PERM_PERMS ${PERM_PERMS} \
     "
 fi
 
 echo "CMD: '$CMD', SECURITY: '$SECURITY', LOCK_ID: '$LOCK', SECURITY_ARGS: '$SECURITY_ARGS'"
 
-PID_FILE=/home/pi/smartlock.pid
+function update_certs {
+  APP_PASSWORD='ZOIWcsPg6BYmLXSv'
```

Comment:
The password should be passed in

Reply:
This script is run by systemctl.  Maybe it should try to read it from a file?

Reply:
@mitza-oci Should we maybe require a password when running `smart-lock pi install` and then read the password from a file on startup?

Reply:
I went ahead and added the password requirement for pi install.

Reply:
I think that will work.